src/client/render.tsx

```diff
@@ -91,7 +91,7 @@ export class CellOutput extends React.Component<ICellOutputProps> {
     }
     private renderLatex(data: nbformat.MultilineString | JSONObject) {
         // Fixup latex to make sure it has the requisite $$ around it
-        data = fixMarkdown(concatMultilineStringOutput(data as nbformat.MultilineString), true);
+        data = fixMarkdown(concatMultilineString(data as nbformat.MultilineString, true), true);
```

Comment:
Merged soem changes from the old renderer into this (some fixes were not brought across)